src/libraries/pretest.proj

```diff
@@ -1,7 +1,19 @@
 <Project Sdk="Microsoft.Build.Traversal">
 
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk" />
+  <PropertyGroup Condition="'$(DotNetBuildTasksSharedFrameworkTaskDir)' == ''">
+    <DotNetBuildTasksSharedFrameworkTaskDir Condition="'$(MSBuildRuntimeType)' == 'core'">$(MSBuildThisFileDirectory)../tools/netcoreapp2.1/</DotNetBuildTasksSharedFrameworkTaskDir>
```

Comment:
The two propertyGroups added here shouldn't be needed. by importing the `Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk` -- `Sdk.props` you should have access to all these properties defined here: https://github.com/dotnet/arcade/blob/1924d7ea148c9f26ca3d82b60f0a775a5389ed22/src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/sdk/Sdk.props#L11